master/internal/rm/kubernetesrm/kubernetes_resource_manager_intg_test.go

```diff
@@ -0,0 +1,242 @@
+//go:build integration
+// +build integration
+
+package kubernetesrm
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	k8sV1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/determined-ai/determined/master/internal/config"
+	"github.com/determined-ai/determined/master/internal/mocks"
+	"github.com/determined-ai/determined/master/pkg/device"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+const (
+	defaultNamespace    = "default"
+	auxNodeName         = "aux"
+	compNodeName        = "comp"
+	pod1NumSlots        = 4
```

Comment:
nit: only use this variable once, so doesn't need to be a const.

Reply:
I use this two places in the test, so for clarity of the test case I'll probably keep this!